### DISCUSSION TITLE
Is there a way to set max-concurrency

### DISCUSSION THREAD

**Discussion by tommasofavaron1** [vllm-project/vllm#2691] [Q&A]
Hi everyone,
I wanted to ask if it is possible to configure a maximum number of concurrent requests.
does it make sense to do this and therefore increase num_requests_waiting or is it just a overhead that doesn't increase the speed of requests in queue?

**Comment by ghbacct** [comment]
Any pointers on this would be greatly appreciated! 🙏

**Comment by csanadpoda** [comment]
Is this what you're looking for? https://docs.vllm.ai/en/v0.6.1.post2/dev/engine/async_llm_engine.html#vllm.AsyncLLMEngine.limit_concurrency
